### PR TITLE
Cleanup prepare-release logs

### PR DESCRIPTION
**Pull request by thegreyd**
Our job logs have considerably increased in length, making them hard to parse through.
- Make the cmd_gather `time elapsed` information debug
- Don't double warn in elliott find-bugs sweep

### CODE REVIEW

File: elliott/elliottlib/cli/find_bugs_sweep_cli.py [hunk 1]
```diff
@@ -368,14 +368,10 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
 
     not_found = set(tracker_bugs) - found
     if not_found:
-        not_found_with_component = [(b.id, b.whiteboard_component) for b in not_found]
-        logger.warning('No attached builds found in advisories for tracker bugs (bug, package):'
-                       f' {not_found_with_component}.')
```
Comment (ashwindasr) [inline elliott/elliottlib/cli/find_bugs_sweep_cli.py:373]: So I guess we don't care about the case in which `not_found` is `True` and `permitted_bug_ids` is false?

If so,
/lgtm

Comment (Ximinhan) [reply elliott/elliottlib/cli/find_bugs_sweep_cli.py:373]: these two lines will log the original `not_found` bug list, after removing the `permitted_bug_ids` is `still_not_found`

Comment (thegreyd) [reply elliott/elliottlib/cli/find_bugs_sweep_cli.py:373]: Yeah, this logging was little bit like a debug thing since I wrote this piece of code - to confirm we are filtering correctly. Now it's unecessary and is noisy to see the same log message twice
